docs/skill_json.md

```diff
@@ -21,18 +22,30 @@ This file contains metadata about the skill and is all you need to submit a skil
     "What is Madonna's real name?",
     "What's 18 times 4?",
     "How many inches in a meter?"
-  ]
+  ],
+  "lang": "en-us",
+  "lang_support": {
```

Comment:
would any language missing here be considered unsupported?

Reply:
I think so, we could do something like with plugins where a missing param assumes full support for backwards-compat? We could also build this dict from skill resource files in automation